deps/chakrashim/src/v8object.cc

```diff
@@ -129,6 +129,12 @@ Maybe<bool> Object::Set(Local<Context> context,
   return Just(true);
 }
 
+Maybe<bool> Object::DefineOwnProperty(
+    Local<Context> context, Local<Name> key, Local<Value> value,
+    PropertyAttribute attributes) {
+    return Set(key, value, attributes, /*force*/false);
```

Comment:
Should this be "/*force*/true"? I assume this function should always call "Object.defineProperty" and cannot simply call "JsSetProperty" as it might already have a setter in place.